modules/schema/schema/session/log_info.schema.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "https://xviz.org/schema/session/log_info.json",
```

Comment:
Looks like we missing a hookup here in the `session.schema.json` file for this type?